packages/lib-classifier/src/components/Classifier/components/Banners/components/AlreadySeenBanner/AlreadySeenBannerContainer.stories.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import styled from 'styled-components'
+import { AlreadySeenBannerContainer } from './AlreadySeenBannerContainer'
+import readme from '../../README.md'
+import backgrounds from '../../../../../../../.storybook/lib/backgrounds'
+import { SubjectFactory } from '../../../../../../../test/factories'
+
+const config = {
+  notes: {
+    markdown: readme
+  }
+}
+
+const darkThemeConfig = Object.assign({}, config, { backgrounds: backgrounds.darkDefault })
+
+const subject = SubjectFactory.build({ already_seen: true })
+
+const StyledBox = styled(Box)`
+  position: relative;
+`
+
+storiesOf('Banners | AlreadySeenBannerContainer', module)
```

Comment:
The container / component abstraction doesn't make too much difference outside of the component; I'd simply call it `AlreadySeenBanner`